readme.md

```diff
@@ -115,7 +146,8 @@ Write some basic CSS that demonstrates changing a CSS property when the device w
 How is rebase different than a merge?
 
 Your Answer:
-```text
+```
+Rebase moves the entire feature branch to begin on the tip of the master branch and thereby incorporates all the new commits into master.  Rebase basically destroys past commits as they are added which creates a very clean project history, while merge ties together the history of feature branches with the master branch resulting in a tree-like structure.
 
```

Comment:
A rebase takes a set of commits, and re-writes them (modifies them) so that they appear to apply to a more recent point in history. This effectively makes it look like the rebased work was originally done on the commit it was rebased on, not the original commit it really was made on. In other words, the history appears to be liner, one change happening at a time, as opposed to being branched off.
 
A merge makes a new commit that describes how to combine the changes made in more than one branch. This preserves the history of how the work was done, but does lead to branches in your history.

A branch should never be rebased if the branch has been shared with others, as it can lead to confusion and conflicts.